SpatialGDK/Source/SpatialGDKFunctionalTests/Public/SpatialFunctionalTest.h

```diff
@@ -331,6 +331,19 @@ class SPATIALGDKFUNCTIONALTESTS_API ASpatialFunctionalTest : public AFunctionalT
 	int GetNumExpectedServers() const { return NumExpectedServers; }
 	void DeleteActorsRegisteredForAutoDestroy();
 
+	// Force Actors having the given tag to migrate an gain authority on the given worker. All server workers must declare
+	// the same delegation at the same time, so we highly recommend that you use the AddStepSetTagDelegation() instead.
+	void SetTagDelegation(FName Tag, int32 ServerWorkerId);
```

Comment:
Would it make sense to just directly move this to private?

Reply:
Ah, I see, the Test that tests the interface accesses this directly

Reply:
I decided to leave it like that if you'd want to do it manually, because some times you have complicated uses like the test done for delegation. If I make them private, that test has to be rewriten and broken up into more steps.